### ISSUE TITLE
Don't display link info box when a link has just been added

### ISSUE THREAD

**Issue by jessallison** [icelab/formalist-standard-react#90]
When adding a link within body text using the rich text editor, after I click 'Save link' there's another link info box that can only be closed by hovering your mouse elsewhere - it’s a little confusing as it looks like the cross would close it (but that removes the link you’ve just added).

I think it would be better if this only appeared when you re-highlight the link text.

**Comment by makenosound** [comment]
Yeah, I see where you’re coming from. The potential conflict here is that we don’t want to force users to have to reselect text unnecessarily, which is what we’d be doing if we remove the selection (or even just hide the popup). That’s not necessarily annoying, but if you want to add a link and then italicise it it would be.

Would a reasonable solution be to more clearly label the `×` button with some text — `Remove` perhaps?

**Comment by jessallison** [comment]
Yep, good thinking — `Remove` explanation or similar beside the cross would do the trick.
